src/jit/compiler.h

```diff
@@ -10321,7 +10321,7 @@ extern BasicBlock dummyBB;
 //      __node: a GenTree*, already declared, that gets updated with each node in the statement, in execution order
 
 #define foreach_treenode_execution_order(__node, __stmt)                                                               \
-    for ((__node) = (__stmt)->gtStmt.gtStmtList; (__node); (__node) = (__node)->gtNext)
+    for ((__node) = (__stmt)->gtStmtList; (__node); (__node) = (__node)->gtNext)
```

Comment:
Looks like this macro has only one use. I'd get rid of it. Bleah.

Reply:
Agree, done.